### ISSUE TITLE
How to get the content of a .docx file as string.

### ISSUE THREAD

**Issue by manuhinCCS** [pnp/pnpjs#2655]
Hi Team,

Can you please help me to get the .docx file content in a readable string format.

### Category
- [x] Question

### Version

Please specify what version of the library you are using: [        "@pnp/sp": "^3.10.0"     ]

Please specify what version(s) of SharePoint you are targeting: 
+-- @microsoft/generator-sharepoint@1.16.1
+-- gulp@4.0.2
+-- npm@6.14.17
+-- yarn@1.22.19
`-- yo@4.3.1

### Expected / Desired Behavior / Question
Response should be in a readable string format.

### Observed Behavior
![image](https://user-images.githubusercontent.com/112081775/234655584-e6c46748-e2a2-4f93-9e6d-f1d84ff885eb.png)

### Steps to Reproduce
![image](https://user-images.githubusercontent.com/112081775/234655877-675418f7-c912-4abb-bcfb-4badaaf1af6b.png)

Thank you for your feedback!

**Comment by bcameron1231** [comment]
You are going to need to use a dedicated library to read the binary content as these types of Office files are encoded specifically for word.

I've used docxTemplater before. They have methods for pulling out text from office. documents.
https://github.com/open-xml-templating/docxtemplater

Additionally, I have found the following library in NPM, but I haven't tried it.
https://www.npmjs.com/package/any-text
